### DISCUSSION TITLE
Convert an anonymous account to a normal account using phone Auth

### DISCUSSION THREAD

**Discussion by ursrudra** [appwrite/appwrite#5185] [Q&A]
I am using phone authentication. initially, I want to allow the users to use it anonymously, but for a few of the features I may ask them to log in, so once they log in with their phone number, I want to link these accounts

**Comment by stnguyen90** [comment]
@ursrudra, thanks for creating this! You should be able to call the [Update Phone API](https://appwrite.io/docs/client/account?sdk=web-default#accountUpdatePhone) to convert the Anonymous User to a regular User.

**Reply by ursrudra** [reply]
> @ursrudra, thanks for creating this! You should be able to call the [Update Phone API](https://appwrite.io/docs/client/account?sdk=web-default#accountUpdatePhone) to convert the Anonymous User to a regular User.

@stnguyen90 Thank you for the response, on the Update Phone API, the Password is marked as a required field, I want to update the phone number and verify it. Is there any way to achieve this?
